include/pros/llemu.hpp

```diff
@@ -114,6 +114,24 @@ bool print(std::int16_t line, const char* fmt, Params... args) {
  */
 bool set_text(std::int16_t line, std::string text);
 
+/**
+ * Set text align type on the emulated three-button LCD screen.
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO  - The LCD has not been initialized. Call lcd_initialize() first.
+ * EINVAL - The line number specified is not in the range [0-7]
+ *
+ * \param line
+ *        The line on which to set align type [0-7]
+ * \param align
+ *        The align type
+ *
+ * \return True if the operation was successful, or false otherwise, setting
+ * errno values as specified above.
+ */
+bool set_text_align(std::int16_t line, lcd_text_align_e_t align);
```

Comment:
the `lcd_text_align_e_t` here should be qualified as `pros::lcd_text_align_e_t`

Reply:
those are all C headers (.h), not C++ header (.hpp)

Reply:
it's possible that I'm misremembering how namespaces work though. full qualification ~may not be~ is probably not necessary when inside the same namespace. if you add it and there's a compiler warning or error, then it's fine to just leave as-is

Reply:
Yeah, they are inside the same namespace. File `adi.hpp` has the same style too. Also, I compiled with 😏 before and there's no warning or error. So that's fine.